reference/tools/cmake/cmaketoolchain.rst

```diff
@@ -451,6 +451,8 @@ CMakeToolchain is affected by these ``[conf]`` variables:
 - **tools.cmake.cmaketoolchain:toolchain_file** user toolchain file to replace the ``conan_toolchain.cmake`` one.
 - **tools.cmake.cmaketoolchain:user_toolchain** list of user toolchains to be included from the ``conan_toolchain.cmake`` file.
 - **tools.android:ndk_path** value for ``ANDROID_NDK_PATH``.
+- **tools.android:cmake_legacy_toolchain**: value for ``ANDROID_USE_LEGACY_TOOLCHAIN_FILE``. It may be useful to set this to ``False`` if compiler flags 
```

Comment:
is it `True` by default? (if you're saying setting it to `False` is useful, I think you imply default True).
if so, why something considered legacy is activated by default?